<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceViewer</class>
 <widget class="QWidget" name="DeviceViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DeviceViewer</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QListWidget" name="devices"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>6</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0" colspan="7">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Memory</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QTableWidget" name="memoryTable">
           <property name="sortingEnabled">
            <bool>false</bool>
           </property>
           <property name="cornerButtonEnabled">
            <bool>true</bool>
           </property>
           <column>
            <property name="text">
             <string>Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Number</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>PCM name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Start sector</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>End sector</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="6">
       <widget class="QPushButton" name="newBtn">
        <property name="text">
         <string>New file</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="7">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Nodes</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QTableWidget" name="nodesTable">
           <column>
            <property name="text">
             <string>Number</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Info</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0" colspan="7">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Other params</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QTableWidget" name="otherTable">
           <column>
            <property name="text">
             <string>key</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>value</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="5">
       <widget class="QPushButton" name="loadBtn">
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4" colspan="3">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="serialNumEdit">
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="nameEdit"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Serial number:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QPushButton" name="saveBtn">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
